components/math/percentageChange.js

```diff
@@ -0,0 +1,28 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.math.percentageChange = function() {
+
+        var percentageChange = function(data) {
+
+            if (data.length === 0) {
+                return [];
+            }
+
+            var initialIndex = percentageChange.initialIndex.value(data);
+            var initialValue = percentageChange.inputValue.value(data[initialIndex]);
+
+            return data.slice(initialIndex, data.length)
```

Comment:
I don't think `percenChange` should perform a slice. Leave that to the user - they might want to produce a chart like this:

http://www.financialsense.com/sites/default/files/users/u244/images/2011/0419-business-cycles.gif